query_execution/QueryExecutionMessages.proto

```diff
@@ -0,0 +1,33 @@
+//   Copyright 2015-2016 Pivotal Software, Inc.
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+syntax = "proto2";
+
+package quickstep.serialization;
+
+// Used for both Normal WorkOrders and RebuildWorkOrders.
+message WorkOrderCompletionMessage {
```

Comment:
I get that the scheduler may want to know some run-time info, and the `worker_id` (or the `shiftboss_id` in near future) in `WorkOrderCompletionMessage `, is good enough to know where the hash table lives.

Even if I miss some info requested by the scheduler here, we could create another PR to improve, right? Note that the goal of this PR is just to replace `ForemanMessage`s with `proto`.